piquasso/_math/fock.py

```diff
@@ -381,8 +347,7 @@ def get_linear_fock_operator(
 
         for index, mode in enumerate(modes):
             operator = self.get_single_mode_squeezing_operator(
-                r=singular_values[index],
-                phi=0.0,
+                r=singular_values[index], phi=0.0
```

Comment:
This change is not needed.

Reply:
As it turns out, this was automatically done by black, because I reformatted it, and after running black it went back to one line again. I assume I should leave it like this then.

Reply:
No, you just have to put back the comma as well.